students/markusiwerbo/final_project_code/final_project_code.ino

```diff
@@ -0,0 +1,102 @@
+#include <LiquidCrystal.h>
+
+char dist[4];   //Character array for serial input 
+int nbytes = 0;   
+
+const int numReadings = 3;  // Last three reading are averaged to account for bad measurements 
```

Comment:
FYI, I bet a median would be a better choice if you're trying to eliminate spurious bad measurements.
E.g., if you get [14, 99, 14], and the 99 is suspect, the average would give you 75.7, but the median would give you the (probably correct) 14.